CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 ## Unreleased
 
 * [CHANGE] Minimum required Go version is now 1.16.
+
+## 1.12.2 / 2022-01-29
```

Comment:
Oops, the date is wrong.

Reply:
I have fixed it in the PR for the main. It was too late for the branch :/